viewer/packages/pointclouds/src/PotreeNodeWrapper.ts

```diff
@@ -82,16 +87,22 @@ export class PotreeNodeWrapper {
     return this._classification;
   }
 
-  pick(renderer: THREE.WebGLRenderer, camera: THREE.Camera, ray: THREE.Ray): PickPoint | null {
-    return this.octree.pick(renderer, camera, ray, { pickWindowSize: 20 });
+  get stylableObjectAnnotationIds(): Iterable<AnnotationMetadata> {
+    return this._annotations.map(a => {
```

Comment:
Why can't it just return `this._annotations` directly? `map()` will allocate. Seems we can get by with only `stylableObjects` below.

Reply:
Because this method is the one being used to expose the annotations to the user. I have made the choice to only expose the annotation ID and the asset ID, the objects in the `_annotations` list also contain e.g. `objectId` and the geometry information, which I have deemed irrelevant to the user. Do you think we should expose more (apart from a bounding box, which will be a future addition)? On another note, the `_annotations` objects` type is a superset of `AnnotationMetadata`, so we could just return the `_annotations` list anyway, but it would contain more data than just what the type insinuates.